src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -147,6 +149,28 @@ const ContentMessageComponent: React.FC<ContentMessageProps> = ({
   const reactionGroupedByUser = groupByReactionUsers(reactions);
   const reactionsTotalCount = Array.from(reactionGroupedByUser).length;
 
+  const isUserRemovingReaction = (reaction: ReactionType): boolean => {
+    const reactions = message.reactions();
+    const userId = selfId.id;
+    const userReactions = reactions[userId] || '';
```

Comment:
This variable should be sufficient

```suggestion
    const userReactions = reactions[userId];
```

Reply:
i want to keep it this way, this way we can ensure to initialise it correctly